src/content/docs/apm/agents/nodejs-agent/installation-configuration/install-nodejs-agent-docker.mdx

```diff
@@ -29,7 +29,12 @@ With just a few additions your existing Dockerfile can be used with our Node.js
 
    Install a specific version, or use any of the other options provided by the [`package.json` format](https://docs.npmjs.com/files/package.json#dependencies). Check [the Node.js agent release notes](/docs/release-notes/agent-release-notes/nodejs-release-notes) for information about past agent versions.
 
-2. Depending on how your container is setup, you can edit the `ENTRYPOINT` to include `newrelic` module first with Node.js `-r`/`--require` flag by running `node -r newrelic YOUR_PROGRAM.js`. If you can't control how your program runs, you can load the `newrelic` module before any other module in your program by adding `require('newrelic')`.
+2. Depending on how your container is set up, you can edit the `ENTRYPOINT` to include `newrelic` module first with Node.js `-r`/`--require` flag by running ```node -r newrelic YOUR_PROGRAM.js```
```

Comment:
Step 2 with all the tips and notes can be adjusted with some tech writer copy edits. Is this step directing the agent to initialize when the container with app image spins up? I might rework it to read something like this:

> 2. Depending on your container set-up, you can adjust where the Node.js flag appears. For example, you can initialize the agent when the container spins up by including the New Relic module ahead of your app code:
>
> ```
> node -r newrelic YOUR_PROGRAM.js
> ```
> ```
> ENTRYPOINT ['node', '-r', 'newrelic', 'YOUR_PROGRAM.js']
> ```
> If you can't control how your program runs, load the New Relic module before any other modules within your program: 
>
>```
> some example code or whatever
> something something require('newrelic')
>```
> If an npm script like `npm start` runs your program, then modify this script by running this instead:
> ```
> npm pkg set scripts.start="node -r newrelic your-program.js"
> ```
> Keep in mind that if your Node.js agent is older than [v7.2.0](https://deploy-preview-18896--docs-website-netlify.netlify.app/docs/release-notes/agent-release-notes/nodejs-release-notes/node-agent-7-2-0/), you will need to add the environment variable `NEW_RELIC_NO_CONFIG_FILE=true` to your Dockerfile so the agent can run without a configuration file. More information on our configuration settings and order of precedence can be found [here](https://deploy-preview-18896--docs-website-netlify.netlify.app/docs/apm/agents/nodejs-agent/installation-configuration/nodejs-agent-configuration/).

Reply:
For the two entry point examples, we can make those child options for the examples

Reply:
I have a version I think is better for these; I'll be pushing that momentarily.